windows/src/engine/keyman32/appint/aiWin2000Unicode.cpp

```diff
@@ -198,7 +198,7 @@ BOOL AIWin2000Unicode::PostKeys()
      (Queue[0].ItemType != QIT_VSHIFTDOWN && Queue[0].ItemType != QIT_VSHIFTUP))   /* I1813 - need to test queuesize as well */
   {
     /* Fakes a key to block Alt being recognised as a menu key */
-    keybd_shift(pInputs, &n, 0, TRUE, FALSE);
+    keybd_shift(pInputs, &i, FALSE);
```

Comment:
This line is the primary bug fix; the rest is code cleanup and debug improvements